README.md

```diff
@@ -407,17 +412,17 @@ The Deck tag is the root level tag for your presentation. It supports the follow
 
 | Name                    | PropType          | Description                                                                                                                                                                   |
```

Comment:
Would it be useful to just add a `Default value` column to this chart instead of wrapping it into the description?